projects/components/src/table/data/table-cdk-data-source.ts

```diff
@@ -59,7 +59,7 @@ export class TableCdkDataSource implements DataSource<TableRow> {
       .pipe(
         tap(() => this.loadingStateSubject.next({ loading$: NEVER })),
         /**
-         * Below debouce is needed to handle multiple emission from buildChangeObservable.
+         * Below debounce is needed to handle multiple emission from buildChangeObservable.
```

Comment:
I assume this is the real reason for the PR and all this other stuff is just because you were working in the area.

Reply:
Yep. Totally